components/board/odroid/data.go

```diff
@@ -12,28 +12,28 @@ var boardInfoMappings = map[string]genericlinux.BoardInformation{
 	// pwm chip is included in the board mapping for simplicity.
 	c4: {
 		PinDefinitions: []genericlinux.PinDefinition{
-			{Name: "3", DeviceName: "gpiochip1", LineNumber: 83},
-			{Name: "5", DeviceName: "gpiochip1", LineNumber: 84},
+			{Name: "3", DeviceName: "gpiochip1", LineNumber: 83, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "5", DeviceName: "gpiochip1", LineNumber: 84, PwmChipSysfsDir: "", PwmID: -1},
 			{Name: "7", DeviceName: "gpiochip1", LineNumber: 71, PwmChipSysfsDir: "ffd1a000.pwm", PwmID: 0},
-			{Name: "8", DeviceName: "gpiochip1", LineNumber: 78},
-			{Name: "10", DeviceName: "gpiochip1", LineNumber: 79},
+			{Name: "8", DeviceName: "gpiochip1", LineNumber: 78, PwmChipSysfsDir: "", PwmID: -1},
+			{Name: "10", DeviceName: "gpiochip1", LineNumber: 79, PwmChipSysfsDir: "", PwmID: -1},
```

Comment:
...I don't get it. Aren't these the default values anyway? How does this PR change anything? The old way looked right to me.

Reply:
Olivia and I discussed: If the PwmID is not -1, the pin is marked as supporting hardware PWM, and that's when the trouble starts. Maybe that should be cleaned up elsewhere, but this PR at least makes things consistent with other boards.